### ISSUE TITLE
No sound from rtsp when push from rtmp

### ISSUE THREAD

**Issue by masudayasuto** [aler9/rtsp-simple-server#942]
## Which version are you using?

v0.18.3

## Which operating system are you using?

- [x] Linux amd64 standard
- [x] Linux amd64 Docker
- [ ] Linux arm64 standard
- [ ] Linux arm64 Docker
- [ ] Linux arm7 standard
- [ ] Linux arm7 Docker
- [ ] Linux arm6 standard
- [ ] Linux arm6 Docker
- [ ] Windows amd64 standard
- [ ] Windows amd64 Docker (WSL backend)
- [ ] macOS amd64 standard
- [ ] macOS amd64 Docker
- [ ] Other (please describe)

## Describe the issue

No sound in RTSP when delivered from RTMP

## Describe how to replicate the issue

I'm sorry, I'm Japanese and I'm not good at English, so I'm using translation.

1. start the server (linux)
2. Publish RTMP stream on OBS(rtmp://192.168.1.231/live/test)
3. Read RTSP stream with VLC player ( no sound )
4. (But if I watch hls in a browser, I can hear the sound)

[Screen Record](https://youtu.be/sy033mP9iz8)

## Did you attach the server logs?

```
$ ./rtsp-simple-server
2022/05/11 00:47:51 INF rtsp-simple-server v0.18.3
2022/05/11 00:47:51 INF [RTSP] listener opened on :8554 (TCP), :8000 (UDP/RTP), :8001 (UDP/RTCP)
2022/05/11 00:47:51 INF [RTMP] listener opened on :1935
2022/05/11 00:47:51 INF [HLS] listener opened on :8888
2022/05/11 00:47:51 INF [RTMP] [conn 192.168.1.233:53136] opened
2022/05/11 00:47:52 INF [RTMP] [conn 192.168.1.233:53136] is publishing to path 'live/test', 2 tracks
2022/05/11 00:47:52 INF [HLS] [conn 192.168.1.233:53145] GET /live/test/index.m3u8
2022/05/11 00:47:52 INF [HLS] [muxer live/test] created
2022/05/11 00:47:52 INF [HLS] [muxer live/test] is converting into HLS
2022/05/11 00:47:52 INF [HLS] [conn 192.168.1.233:53145] GET /live/test/stream.m3u8
2022/05/11 00:47:59 INF [HLS] [conn 192.168.1.233:53179] GET /live/test/
2022/05/11 00:47:59 INF [HLS] [conn 192.168.1.233:53179] GET /live/test/index.m3u8
2022/05/11 00:47:59 INF [HLS] [conn 192.168.1.233:53179] GET /live/test/stream.m3u8
2022/05/11 00:48:09 INF [HLS] [conn 192.168.1.233:53179] GET /live/test/1652230080.ts
2022/05/11 00:48:17 INF [HLS] [conn 192.168.1.233:53179] GET /live/test/stream.m3u8
2022/05/11 00:48:17 INF [HLS] [conn 192.168.1.233:53179] GET /live/test/1652230089.ts
2022/05/11 00:48:17 INF [HLS] [conn 192.168.1.233:53179] GET /live/test/1652230080.ts
2022/05/11 00:48:17 INF [HLS] [conn 192.168.1.233:53179] GET /live/test/1652230089.ts
2022/05/11 00:48:17 INF [HLS] [conn 192.168.1.233:53179] GET /live/test/1652230080.ts
2022/05/11 00:48:17 INF [HLS] [conn 192.168.1.233:53179] GET /live/test/1652230089.ts
2022/05/11 00:48:17 INF [HLS] [conn 192.168.1.233:53179] GET /favicon.ico
2022/05/11 00:48:25 INF [HLS] [conn 192.168.1.233:53179] GET /live/test/stream.m3u8
2022/05/11 00:48:25 INF [HLS] [conn 192.168.1.233:53179] GET /live/test/1652230097.ts
2022/05/11 00:48:34 INF [HLS] [conn 192.168.1.233:53179] GET /live/test/stream.m3u8
2022/05/11 00:48:34 INF [HLS] [conn 192.168.1.233:53179] GET /live/test/1652230105.ts
2022/05/11 00:48:39 INF [RTSP] [conn 192.168.1.233:57461] opened
2022/05/11 00:48:39 INF [RTSP] [session 672451322] created by 192.168.1.233:57461
2022/05/11 00:48:39 INF [RTSP] [session 672451322] is reading from path 'live/test', 2 tracks with UDP
2022/05/11 00:48:42 INF [HLS] [conn 192.168.1.233:53179] GET /live/test/stream.m3u8
2022/05/11 00:48:42 INF [HLS] [conn 192.168.1.233:53179] GET /live/test/1652230114.ts
2022/05/11 00:48:42 INF [HLS] [conn 192.168.1.233:53179] GET /live/test/1652230105.ts
2022/05/11 00:48:50 INF [HLS] [conn 192.168.1.233:53179] GET /live/test/stream.m3u8
2022/05/11 00:48:50 INF [HLS] [conn 192.168.1.233:53179] GET /live/test/1652230122.ts
2022/05/11 00:48:59 INF [HLS] [conn 192.168.1.233:53179] GET /live/test/stream.m3u8
2022/05/11 00:48:59 INF [HLS] [conn 192.168.1.233:53179] GET /live/test/1652230130.ts
2022/05/11 00:49:07 INF [HLS] [conn 192.168.1.233:53179] GET /live/test/stream.m3u8
2022/05/11 00:49:07 INF [HLS] [conn 192.168.1.233:53179] GET /live/test/1652230139.ts
^C2022/05/11 00:49:09 INF shutting down gracefully
2022/05/11 00:49:09 INF [RTMP] [conn 192.168.1.233:53136] closed (terminated)
2022/05/11 00:49:09 INF [RTSP] [conn 192.168.1.233:57461] closed (terminated)
2022/05/11 00:49:09 INF [RTSP] [session 672451322] destroyed (terminated)
2022/05/11 00:49:09 INF [HLS] [muxer live/test] destroyed (terminated)
2022/05/11 00:49:09 INF [RTSP] listener is closing
2022/05/11 00:49:09 INF [HLS] listener is closing
2022/05/11 00:49:09 INF [RTMP] listener is closing
2022/05/11 00:49:09 INF waiting for external commands
```

## Did you attach a network dump?

no

**Comment by rassi0429** [comment]
It's likely my problem, but I'll provide you with the information you need. Thank you

**Comment by aler9** [comment]
Hello, this is probably a duplicate of #932 - please try this nightly release in order to test whether the bug has been fixed:
https://github.com/aler9/rtsp-simple-server/issues/932#issuecomment-1126205794

**Comment by masudayasuto** [comment]
@aler9 Thank you for your reply. I tried the beta build version and it sounded fine.

Thank you very much. 
Close the issue

**Comment by aler9** [comment]
fixed in v0.18.4.
